wunderio/drupal-project-k8s
# Different secret name

Use different secret name for custom certificates.

Testing:
1. Deploy environment with standard settings, plus extra domain via exposeDomains - both must be using LetsEncrypt
2. Switch domain in exposeDomains to custom, append the certificate from command below.
3. After deployment, domain should be using the custom certificate. 


Creating self-signed certificate
`openssl req -newkey rsa:4096 \
            -x509 \
            -sha256 \
            -days 3650 \
            -nodes \
            -out def_domain.crt \
            -keyout def_domain.key \
            -subj "/C=LV/ST=Test/L=Test/O=Dev/OU=IT Department/CN=<domain here>"`
